SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/EmailConfirmationJob.swift

```diff
@@ -128,7 +128,16 @@ public class EmailConfirmationJob: Operation, @unchecked Sendable {
 
         let extractedProfile = extractedProfileData.profile
 
+        let attemptId: UUID
+        if let parsed = UUID(uuidString: jobData.attemptID) {
+            attemptId = parsed
+        } else {
+            assertionFailure("Email confirmation job stored attemptID must be a valid UUID")
+            attemptId = UUID()
```

Comment:
What’s the impact of generating a random attempt ID here (in terms of observability)?

Reply:
It’ll be detached from the earlier halted opt-out job. We want to have both jobs included in the wide event.